services/gundeck/src/Gundeck/Push/Native/Serialise.hs

```diff
@@ -77,23 +76,20 @@ renderText t aps prio x = case t of
         [ "aps" .= object [],
           "data" .= x
         ]
+    -- https://developer.apple.com/documentation/usernotifications/modifying_content_in_newly_delivered_notifications
+    -- Must contain `mutable-content: 1` and  include an alert dictionary with title, subtitle, or body information.
+    -- Since we have no useful data here, we send a default payload that gets overriden by the client
     apsDict HighPriority =
-      object $
-        "alert"
-          .= object
-            ( "loc-key" .= (aps ^? _Just . apsLocKey)
-                # "loc-args" .= (aps ^? _Just . apsLocArgs)
-                # []
-            )
-          # "sound"
-          .= (aps ^? _Just . apsSound)
-          # "content-available"
-          .= '1'
-          # []
+      object
+        [ "alert" .= object ["title" .= ("New message" :: Text)],
+          "mutable-content" .= '1'
+        ]
     apsDict LowPriority =
-      object $
```

Comment:
`HighPriority` and `LowPriority` are handled identically, and there are no other constructors, so this argument should not be passed here.